tasks/main.yml

```diff
@@ -64,6 +64,10 @@
     ignore_errors: "{{ network_ignore_errors | default(omit) }}"
     force_state_change: "{{ network_force_state_change | default(omit) }}"
     connections: "{{ network_connections | default([]) }}"
+  register: network_connections_result
```

Comment:
Another thought, can you please rename the variable to `__network_connections_result` to indicate (for now), that it is internal? This makes it more clear that we do not promise anything about it.

Reply:
Ok, fixed!